salt/_pillar/metalk8s.py

```diff
@@ -78,12 +79,34 @@ def _load_apiserver(config_data):
     return result
 
 
+def _load_iso_path(config):
+    """Load iso path from BootstrapConfiguration
+
+    """
+    res = {}
+
+    for product in config['products']['metalk8s']:
+        if __salt__['file.directory_exists'](product):
+            version = __salt__['metalk8s.product_version_from_tree'](product)
```

Comment:
Do we really want to support this ? For me we don't.
Any news about this @gdemonet or can I remove it ?

Reply:
Yes we need it, done